### PR TITLE
feature/DTONBWTWO-702 update ios target version

### PR DESCRIPTION
**Pull request by fmattos-hw**

### CODE REVIEW

File: HyperwalletUISDK.podspec [hunk 1]
```diff
@@ -6,18 +6,18 @@ Pod::Spec.new do |spec|
     spec.license               = { :type => 'MIT', :file => 'LICENSE' }
```
Comment (jsaini-hw) [inline HyperwalletUISDK.podspec:6]: You will have to update `spec.version               = '1.0.0-beta16'` as well

Comment (fmattos-hw) [reply HyperwalletUISDK.podspec:6]: it's already updated https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/blob/f9901ebd6dc44d9e3db8ebe1335ca8bab2fe7271/HyperwalletUISDK.podspec

Comment (jsaini-hw) [reply HyperwalletUISDK.podspec:6]: I am talking about the new version that you will release of UI SDK. You need to update the new version for cocoapods as well

Comment (fmattos-hw) [reply HyperwalletUISDK.podspec:6]: got it, updated to version beta 17

**Pull request merged**